index.js

```diff
@@ -1,17 +1,20 @@
 import { Builder, Browser, By, until } from 'selenium-webdriver';
 import firefox from 'selenium-webdriver/firefox.js';
+import chrome from 'selenium-webdriver/chrome.js';
 import fs from 'fs';
-import { downloadAddon, sleep, switchToWindowWithUrl } from './helpers.js';
+import { downloadAddon, sleep, switchToWindowWithUrl } from './src/helpers.js';
 
 const timestamp = new Date().toISOString();
+const isRegionEU = Boolean(process.argv.find((arg) => arg === '--EU'));
```

Comment:
lets not move them as they pollute the diff

Reply:
One line has been added above as far as I can see. Webdriver for Chroma.